doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -318,6 +318,20 @@ Bluetooth Fast Pair samples
   * The values for the :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option in all configurations, and for the :kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN_TX_POWER_CORRECTION_VAL` Kconfig option in configurations with the Find My Device Network (FMDN) extension support.
     The values are now aligned with the Fast Pair requirements.
 
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Added:
```

Comment:
Consider adding an entry regarding the integration with Application versioning management and the `VERSION` file. You can probably reuse the changelog entry in some way that I added previously for the nRF Desktop